examples/visualisation/visualise_data.ipynb

```diff
@@ -19,13 +21,64 @@
     "### `visualization`\n",
     "contains utilities to draw additional information (e.g. trajectories) onto RGB images. These utilities are commonly used after a `to_rgb` call to add other information to the final visualisation. \n",
     "One example is:\n",
-    "- `draw_trajectory`: this function draws 2D trajectories from coordinates and yaws offset on an image\n"
+    "- `draw_trajectory`: this function draws 2D trajectories from coordinates and yaws offset on an image\n",
+    "\n",
+    "*By using L5 datasets, you accept the license at https://self-driving.lyft.com/level5/download/ and the licenses present inside of each dataset archive.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "-2bOYHKU1tw2"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Download L5 Sample Dataset and install L5Kit\n",
+    "!rm -rf l5kit_dataset\n",
+    "\n",
+    "# Download sample zarr\n",
+    "print(\"Downloading sample zarr dataset...\")\n",
+    "!wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/sample.tar -q --show-progress\n",
+    "!tar xf sample.tar\n",
+    "!rm -rf sample.tar\n",
+    "!mkdir -p l5kit_dataset/scenes\n",
+    "!mv sample.zarr l5kit_dataset/scenes\n",
+    "\n",
+    "# Download semantic map\n",
+    "print(\"Downloading semantic map...\")\n",
+    "!wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/semantic_map.tar -q --show-progress\n",
+    "!mkdir -p l5kit_dataset/semantic_map\n",
+    "!tar xf semantic_map.tar -C l5kit_dataset/semantic_map\n",
+    "!rm -rf semantic_map.tar\n",
+    "\n",
+    "# Download aerial maps\n",
+    "print(\"Downloading aerial maps (this can take a while)...\")\n",
+    "!wget https://lyft-l5-datasets-public.s3-us-west-2.amazonaws.com/prediction/v1.1/aerial_map.tar -q --show-progress\n",
+    "!tar xf aerial_map.tar\n",
+    "!mv aerial_map l5kit_dataset\n",
+    "!rm -rf aerial_map.tar\n",
+    "\n",
+    "# Dowload sample configuration\n",
+    "!wget https://raw.githubusercontent.com/lyft/l5kit/master/examples/visualisation/visualisation_config.yaml -q\n",
+    "\n",
+    "# Install L5Kit\n",
+    "print(\"Installing L5kit...\")\n",
+    "!pip install --progress-bar off --quiet -U l5kit pyyaml\n",
```

Comment:
`pyyaml` should already been pulled from `l5kit` I think..

Reply:
I think this is an issue w/ Colab, because it required to have this last version work there.